README.md

```diff
@@ -3,6 +3,12 @@
 
 ## Contributing to this Repository
 
+* Error: Permission denied (publickey info
```

Comment:
Nice, great link. I remember we had this issue before. What do you think of adding back the parenthesis here to make the verbiage consistent: 
```suggestion
* Error: Permission denied (publickey info)
```